lib/zip/version.rb

```diff
@@ -1,3 +1,3 @@
 module Zip
-  VERSION = '1.2.1'
+  VERSION = '2.0.0'
```

Comment:
@jdleesmiller can you please revert this? Or at least release a patch version of 1.2 as well?

The thing is all the libraries we have in `Gemfile.lock` either explicitly declare `< 2.0.0` or `~> 1.2`

Reply:
Hmm. That is a good point about version constraints.

My rationale was that dropping support for symlinks completely is a breaking change, and I'd be unhappy if [semver](https://semver.org/) minor or patch release broke my app.

If you have `~> 1.2`, I guess 1.3.0 should do it. It's not very semantic, but it does seem like the pragmatic solution. Happy to try it out.